VCFv4.3.tex

```diff
@@ -173,30 +173,49 @@ \subsubsection{Individual format field format}
 Possible Types for FORMAT fields are: Integer, Float, Character, and String (this field is otherwise defined precisely as the INFO field).
 
 \subsubsection{Alternative allele field format}
-Symbolic alternate alleles are described as follows:
+Symbolic alternate alleles specified in the ALT field are described as follows:
 \begin{verbatim}
 ##ALT=<ID=type,Description=description>
 \end{verbatim}
 
 \noindent \textbf{Structural Variants} \newline
-In symbolic alternate alleles for imprecise structural variants, the ID field indicates the type of structural variant, and can be a colon-separated list of types and subtypes.
-ID values are case sensitive strings and must not contain whitespace or angle brackets.
-The first level type must be one of the following:
-\begin{itemize}
-  \item DEL Deletion relative to the reference
-  \item INS Insertion of novel sequence relative to the reference
-  \item DUP Region of elevated copy number relative to the reference
-  \item INV Inversion of reference sequence
-  \item CNV Copy number variable region (may be both deletion and duplication)
-  \item BND Breakend
-\end{itemize}
 
-The CNV category should not be used when a more specific category can be applied. Reserved subtypes include:
-\begin{itemize}
-  \item DUP:TANDEM Tandem duplication
-  \item DEL:ME Deletion of mobile element relative to the reference
-  \item INS:ME Insertion of a mobile element relative to the reference
-\end{itemize}
+
+\begin{tabular}{l l}
+DEL  &  Deletion relative to the reference \\
+INS  &  Insertion relative to the reference \\
+DUP  &  Tandem duplication relative to the reference \\
+INV  &  Inversion of reference sequence \\
+CNV  &  Copy number variable region \\
+BND  &  Breakend \\
```

Comment:
Is `<BND>` a valid first-level symbolic alt (which I think this language implies) according to this PR, or do breakend alts need to be written in the detailed format with square brackets, etc?

Reply:
@d-cameron may best be able to speak to this, but it is my understanding that the intention when using SVTYPE=BND is that the ALT column always contain the bracketed notation, e.g., "**]5:7576543]G**". @cwhelan, can you provide an (hypothetical) example of a variant in which SVTYPE=BND and ALT=<BND>? I am not sure what that would look like.

Reply:
I'm not suggesting that it should be allowed (although this is something the current release of gnomAD does; I'm working to fix that for future releases right now); just trying to point out that unless I'm misreading something in the language in this PR, the specification would seem to allow it even after these changes are merged.

Reply:
At this stage we're trying to fix the simple SV types before addressing BNDs, so I've separated these discussions into https://github.com/samtools/hts-specs/issues/546 and will resolve this thread